test/com/twu/biblioteca/WelcomeTest.java

```diff
@@ -0,0 +1,25 @@
+package com.twu.biblioteca;
+
+
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class WelcomeTest {
```

Comment:
As a convention (in Java), if you are writing a test for class "X", then your test file should be named "XTest.java" and the class contained inside it should be "XTest" in the same package as the class it is testing.  In your case, you have "WelcomeTest" testing "WelcomeMessage", so there's a bit of dissonance there.

Reply:
Thanks, have updated the names of the test file and test class to be consistent.